### PR TITLE
feat: adds vagrant cloud callout

### PR DESCRIPTION
**Pull request by kendallstrautman**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-ksadd-vagrant-cloud-hashicorp.vercel.app/vagrant) 🔎
- [Asana task](https://app.asana.com/0/1203301812856773/1203483857950104) 🎟️

## 🗒️ What

We missed vagrant cloud in the first callout pass. This adds it to the product landing and both install pages:

- [landing](https://dev-portal-git-ksadd-vagrant-cloud-hashicorp.vercel.app/vagrant)
- [install](https://dev-portal-git-ksadd-vagrant-cloud-hashicorp.vercel.app/vagrant/downloads)
- [vmware install](https://dev-portal-git-ksadd-vagrant-cloud-hashicorp.vercel.app/vagrant/downloads/vmware)

Screen Shot 2022-12-08 at 9 30 57 AM

### CODE REVIEW

File: src/components/try-hcp-callout/content.ts [hunk 1]
```diff
@@ -59,4 +59,11 @@ export const tryHcpCalloutContent: Record<
 		ctaText: 'Try cloud for free',
 		ctaUrl: 'https://portal.cloud.hashicorp.com/sign-up',
 	},
+	vagrant: {
+		heading: 'Vagrant Cloud',
+		description:
```
Comment (kendallstrautman) [inline src/components/try-hcp-callout/content.ts:64]: It seems like the description isn't used by the prebuilt callout anymore, should we remove it from the interface?

Comment (kendallstrautman) [reply src/components/try-hcp-callout/content.ts:64]: nvm, it was fixed in #1421

**Review by zchsh — APPROVED** [review]
✅ Looks great to me! Thanks for this 🙇

**Comment by emilypersson1** [comment]
@kendallstrautman just commented in Asana, but only feedback I have is I'm not seeing it in the header, but looks great everywhere else!

**Comment by braydenlove** [comment]
@kendallstrautman the bg of the ui is appearing to be gray and needs to be white. I am going to DM you a new svg to try as I can't attached images to GitHub still.

**Comment by kendallstrautman** [comment]
@emilypersson1 oh I didn't realize we were adding a link in the nav! I'll add it

**Comment by kendallstrautman** [comment]
[PR here for the 'try cloud' link](https://github.com/hashicorp/dev-portal/pull/1428)

File: src/components/try-hcp-callout/content.ts [hunk 2]
```diff
@@ -59,4 +59,11 @@ export const tryHcpCalloutContent: Record<
...
+			'Virtual boxes for Linux, Laravel and any development environment',
+		ctaText: 'Try Vagrant Cloud',
+		ctaUrl: 'https://app.vagrantup.com/boxes/search',
```
Comment (kendallstrautman) [inline src/components/try-hcp-callout/content.ts:67]: @emilypersson1 should this actually link to https://app.vagrantup.com/account/new? So it matches the other CTA behavior linking to the sign up page?

**Comment by kendallstrautman** [comment]
@braydenlove updated that asset, lmk if that's better!

**Pull request merged**